proto/slinky/marketmap/v1/tx.proto

```diff
@@ -36,11 +39,26 @@ message CreateMarket {
   Paths paths = 3 [ (gogoproto.nullable) = false ];
 }
 
-// MsgUpdateMarketMap defines a message carrying a payload for updating the
-// x/marketmap module.
-message MsgUpdateMarketMap {
+// UpdateMarket contains all information needed to update market.
+message UpdateMarket {
+  // Ticker is the on-chain representation of the ticker. This is the target
+  // ticker that the prices of the set of tickers will be converted to.
+  Ticker ticker = 1 [ (gogoproto.nullable) = false ];
+
+  // Providers maps provider names to their off-chain
+  // representations for the given ticker of the message.
+  Providers providers = 2 [ (gogoproto.nullable) = false ];
+
+  // Paths is the list of convertable markets that will be used to convert the
+  // prices of a set of tickers to a common ticker.
+  Paths paths = 3 [ (gogoproto.nullable) = false ];
+}
```

Comment:
Since this has direct overlap with `CreateMarket` does it make sense to just have a single `Market` struct?